### ISSUE TITLE
Not working with two accoutns

### ISSUE THREAD

**Issue by richhalliwell** [hultenvp/solis-sensor#6]
When i try and use this for two separate accounts (mine and my neighbours) it doesn't seem to work. Occasionally it will work for one and not the other. I suspect it is attempting to use the same session for both and not creating two separate login sessions? Can you confirm my suspicions?

**Comment by hultenvp** [comment]
@richhalliwell Yeah, I think your suspicion might be true. I'm not sure, but I might need to fetch http session only once and store it for further use. I'll try to create a test version for you. I'll ping you when done.

**Comment by hultenvp** [comment]
@richhalliwell Please try the code here: https://github.com/hultenvp/solis-sensor/tree/6-not-working-with-2-accounts/custom_components/solis

**Comment by hultenvp** [comment]
@richhalliwell: Any progress? Does the fix work?

**Comment by hultenvp** [comment]
@richhalliwell : Gentle reminder. Does the fix work?

**Comment by jdpl-ratel** [comment]
If something has changed in this topic and you can now run two instances?

**Comment by hultenvp** [comment]
Each account specified is one instance.
Each instance only supports one Plant, but all inverters in that plant.
